README.md

```diff
@@ -56,7 +56,10 @@ python -m torch.distributed.launch --nproc_per_node=$NGPUS eval.py --model fcn32
 ### Demo
 ```
 cd ./scripts
-python demo.py --model fcn32s_vgg16_voc --input-pic ./datasets/test.jpg
+python demo.py --model fcn32s_vgg16_voc --input-pic ../datasets/test.jpg
+or
```

Comment:
Maybe the `or` should be removed.

Reply:
I add a new line below 'or':
python demo.py --model fcn32s_vgg16_voc --input-pic ../tests/test_img.jpg